### PR TITLE
Xompletions

### PR DESCRIPTION
**Pull request by gforsyth**
Adds completions for `xonfig` and `xontrib`

### CODE REVIEW

File: xonsh/completers/xompletions.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import xonsh.xontribs as xx
```
Comment (scopatz) [inline xonsh/completers/xompletions.py]: Can you add a module docstring, please? Also can you add a sphinx autodoc hook for this module too, please? Thanks!

File: xonsh/completers/xompletions.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+from xonsh.tools import color_style_names
```
Comment (scopatz) [inline xonsh/completers/xompletions.py]: This should be `import xonsh.tools as xt` to get the benefits of lazy module amalgamation.

**Comment by scopatz** [comment]
Thanks @gforsyth! Just a couple of small issues. Really looking forward to this!

**Comment by gforsyth** [comment]
Hey @scopatz -- fixed the issues you raised.  Also an unrelated fix I tacked on for the events docs generation -- was raising a bunch of warnings in Sphinx

**Comment by scopatz** [comment]
Looks great, thanks!

**Pull request merged**